src/components/SlateEditor/plugins/codeBlock/CodeBlock.tsx

```diff
@@ -6,7 +6,7 @@ import he from 'he';
 import Button from '@ndla/button';
 import { Cross } from '@ndla/icons/action';
 import { injectT } from '@ndla/i18n';
-import { Codeblock, getTitleFromFormat } from '@ndla/code';
```

Comment:
kanskje vi kan fjerne hele `getTitleFromFormat` fra frontend-packages nå om den ikke trengs og ikke brukes i article-converter? Eller fjerne export på den.

Reply:
Den brukes vel i frontend-packages, men er ikke noe i veien for å fjerne exporten.